traffic_ops/app/lib/API/Cdn.pm

```diff
@@ -123,10 +126,14 @@ sub create {
 		return $self->alert( "a cdn with domain " . $params->{domainName} . " already exists." );
 	}
 
+	#setting tenant_id to the user's tenant if tenant is not set. TODO(nirs): remove when tenancy is no longer optional in the API
+	my $tenant_id = exists($params->{tenantId}) ? $params->{tenantId} :  $self->current_user_tenant();
+
 	my $values = {
 		name => $params->{name},
 		dnssec_enabled => $params->{dnssecEnabled},
 		domain_name => $params->{domainName},
+		tenant_id => $tenant_id,
```

Comment:
I'm not familiar with this syntax.
I assume it is equivalent to "defined".
The point is, that as "undef" tenant is a valid value that the user may set. Therefore I prefer to use "exists" and not "defined".